packages/syntax-core/src/Box/Box.tsx

```diff
@@ -98,6 +99,15 @@ type BoxProps = {
    * The background color of the box.
    */
   backgroundColor?: (typeof allColors)[number];
+  /**
+   * The border styling of the box.
+   *
+   * * `primary`: 1px solid var(--color-base-primary)
```

Comment:
@somethiiing could we update this to the actual value?

```suggestion
   * * `primary`: 1px solid var(--color-cambio-gray-370)
```